assets/src/scripts/interactive/pages/build-query.jsx

```diff
@@ -33,7 +33,7 @@ function QueryBuilder() {
           if (value === undefined || Number.isNaN(value)) return false;
 
           const { timeScale } = testContext.parent;
-          if (timeScale === "weeks" && value > 522) return false;
+          if (timeScale === "weeks" && value > 520) return false;
```

Comment:
Not all years have 52 weeks, some have 53, so 522 is closer to the number of weeks over a ten year period.

Reply:
TIL